Formula/v2ray.rb

```diff
@@ -0,0 +1,78 @@
+class V2ray < Formula
+  desc "Project V provide a unified platform for anti-censorship"
+  homepage "https://github.com/v2ray/v2ray-core"
+  version "4.22.1"
+  url "https://github.com/v2ray/v2ray-core/archive/v#{version}.tar.gz"
+  sha256 "31c1934eeac3552c7ab68eac9dc3e964e05f3c743b3733b0b6a0159c495019d6"
+
+  depends_on "go" => :build
+  depends_on "geoip"
+
+  resource "geosite" do
+    url "https://github.com/v2ray/domain-list-community/releases/download/202001070832/dlc.dat"
+    sha256 "f58fa04f8c943584a3b3ffc5678f1dc27abc429a9ff600868ba97f313938f2e9"
+  end
+  resource "geoip" do
+    url "https://github.com/v2ray/geoip/releases/download/202001070102/geoip.dat"
+    sha256 "434d3f415655baa2e991a78786defe5dff798f245445cfe4580ac5d6a4a3eecf"
+  end
```

Comment:
Is this different from the geoip dependency above?

Reply:
I just removed GeoIP dependency,  v2ray has an ASSET directory setting, geoip.dat, and geosite.dat   both need put in the directory.  If depends GeoIP, may need additional symbol link to config dir. I think it's not good.